packages/Molecules/Form/package.json

```diff
@@ -16,6 +16,7 @@
     "vue-form-generator": "^2.2.2",
     "vue-multiselect": "^2.1.3",
     "vue-on-click-outside": "^1.0.3",
-    "vue-scrollto": "^2.13.0"
+    "vue-scrollto": "^2.13.0",
+    "vue-media": "^1.1.1"
```

Comment:
@dylankelly Are you using `vue-media` any place? Just couldn't find it. Also we have a `breakpoint` mixins https://github.com/dpc-sdp/ripple/blob/master/packages/Atoms/Global/mixins/breakpoint.js which you may can use for same purpose.

Reply:
ah cool, i hadn't seen that mixin.